portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase

```diff
@@ -588,6 +588,91 @@ definition {
 		}
 	}
 
+	@description = "This is for LPS-130083. View the ellipsis button on the top bar when the fragment is inside the nested grid."
```

Comment:
View the ellipsis button on the topper of fragment when the fragment is inside a nested Grid.